pkgs/applications/video/obs-studio/obs-ndi.nix

```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     description = "Network A/V plugin for OBS Studio";
     homepage = "https://github.com/Palakis/obs-ndi";
-    maintainers = with maintainers; [ peti ];
+    maintainers = with maintainers; [ peti jshcmpbll ];
```

Comment:
Did you add yourself to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix?

Reply:
I did not! Thanks :D